desc/compute/_core.py

```diff
@@ -5,6 +5,28 @@
 from .data_index import register_compute_fun
 
 
+@register_compute_fun(
+    name="NFP",
+    label="N_{FP}",
+    units="~",
+    units_long="None",
+    description="Number of toroidal field periods",
+    dim=0,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="",
+    data=[],
+    parameterization=[
+        "desc.equilibrium.equilibrium.Equilibrium",
+        "desc.magnetic_fields.OmnigenousField",
+    ],
+)
+def _NFP(params, transforms, profiles, data, **kwargs):
+    data["NFP"] = transforms["grid"].NFP
```

Comment:
could this cause issues if the grid and basis have different NFP?

Reply:
I only used this to replace references to `grid.NFP`, so if another compute quantity needs a different basis NFP for some reason it could reference that directly. 

This change isn't necessary, but we have several compute quantities now that reference NFP so I thought it made sense.